airflow/gcp/operators/bigquery.py

```diff
@@ -217,7 +217,17 @@ class BigQueryGetDataOperator(BaseOperator):
     be equal to the number of rows fetched. Each element in the list will again be a list
     where element would represent the columns values for that row.
 
-    **Example Result**: ``[['Tony', '10'], ['Mike', '20'], ['Steve', '15']]``
+    .. seealso:: https://cloud.google.com/bigquery/docs/reference/v2/tabledata/list
+
+    **Example Result**: ::
```

Comment:
```suggestion
    **Example Result**:
```
I am not sure if you need those colons. You can check by building the docs via `./breeze --build-docs` if it makes a difference.

Reply:
It fails if I remove the `::`